src/main/java/com/minecolonies/coremod/blocks/BlockWaypoint.java

```diff
@@ -0,0 +1,98 @@
+package com.minecolonies.coremod.blocks;
+
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.coremod.creativetab.ModCreativeTabs;
+import net.minecraft.block.Block;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.item.ItemBlock;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+
+import java.util.Locale;
+
+import static com.minecolonies.api.util.constant.Suppression.DEPRECATION;
+
+/**
+ * This block is used as a substitution block for the Builder.
+ * Every solid block can be substituted by this block in schematics.
+ * This helps make schematics independent from location and ground.
+ */
+public class BlockWaypoint extends Block
+{
+
+    /**
+     * The hardness this block has.
+     */
+    private static final float BLOCK_HARDNESS = 0.0F;
+
+    /**
+     * This blocks name.
+     */
+    private static final String BLOCK_NAME = "blockwaypoint";
+
+    /**
+     * The resistance this block has.
+     */
+    private static final float RESISTANCE = 1F;
+
+    /**
+     * Constructor for the Substitution block.
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    public BlockWaypoint()
+    {
+        super(Material.WOOD);
+        initBlock();
+    }
+
+    /**
+     * initialize the block
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    private void initBlock()
+    {
+        setRegistryName(BLOCK_NAME);
+        setUnlocalizedName(String.format("%s.%s", Constants.MOD_ID.toLowerCase(Locale.ENGLISH), BLOCK_NAME));
```

Comment:
what does the Locale.English part do? (haven't seen it anywhere else)

Reply:
Nothing as Locale is not set in minecraft, and it then defaults to English.

Reply:
ahh, cool